.github/workflows/deploy.yaml

```
@@ -196,7 +196,16 @@ jobs:
         echo "This PR should only modify non-production files, and it should be auto-approved shortly." >> /tmp/body.txt
         echo "Once it's auto-approved, it should auto-merge." >> /tmp/body.txt
         gh pr create --assignee @me --fill --body-file /tmp/body.txt
-        gh pr checks --watch --required
+        # Avoid a race condition with checks not existing immediately upon PR creation.
+        TRY=1
+        while ! gh pr checks --watch --required ; do
+          sleep 10
+          TRY=$(expr $TRY + 1)
+          if [ $TRY -ge 30 ] ; then
+            echo "Failed to wait for PR checks."
+            exit 1
+          fi
+        done
```

@coltfred I did it that way for brevity and clarity, and it's situational. Here's what it looks like using the other style of retry loop:
```suggestion
        # Avoid a race condition with checks not existing immediately upon PR creation.
        for TRY in $(seq 30) ; do
          if gh pr checks --watch --required ; then
            GH_SUCCESS=true
            break
          else
            GH_SUCCESS=false
            sleep 10
          fi
        done
        if [ "$GH_SUCCESS" = "false" ] ; then
          echo "Failed to wait for PR checks."
          exit 1
        fi
```
I think it looks better with the increment, in this particular context. But I realize that what looks good to me may look bad to others. I'll change it if you think it's easier to read with the `seq 30`.